cmd/tailscale/cli/web.go

```diff
@@ -86,17 +87,20 @@ func runWeb(ctx context.Context, args []string) error {
 		return fmt.Errorf("too many non-flag arguments: %q", args)
 	}
 
-	st, err := localClient.StatusWithoutPeers(ctx)
-	if err != nil {
-		return fmt.Errorf("getting client status: %w", err)
+	var hasPreviewCap bool
+	var selfIP netip.Addr
+	if st, err := localClient.StatusWithoutPeers(ctx); err == nil && st.Self != nil {
```

Comment:
`err == nil && st.Self != nil && len(st.Self.TailscaleIPs) > 0` avoids logging the conditionally set `selfIP` variable